web-frontend/src/pages/Assets/DeleteAssets/DeleteAssetModal.tsx

```diff
@@ -0,0 +1,30 @@
+import { PopupModal, PopupModalHeader } from '@massalabs/react-ui-kit';
+
+import Intl from '@/i18n/i18n';
+import { ConfirmDelete } from '@/pages/Assets/DeleteAssets/ConfirmDelete';
+
+interface DeleteAssetModal {
+  setModalOpen: (value: boolean) => void;
+  tokenAddress: string;
+}
+
+export function DeleteAssetModal({ ...props }: DeleteAssetModal) {
+  const { setModalOpen } = props;
+
+  return (
+    <PopupModal
+      customClass="!w-[440px] h-1/2"
+      fullMode={true}
+      onClose={() => setModalOpen(false)}
+    >
+      <PopupModalHeader>
+        <div className="flex flex-col mb-4">
+          <div className="mas-title mb-4">{Intl.t('assets.delete.title')}</div>
+
+          <div className="mas-body2">{Intl.t('assets.delete.subtitle')}</div>
```

Comment:
```suggestion
          <p className="mas-body2">{Intl.t('assets.delete.subtitle')}</p>
```

Same here! If it's a body value, you can place a `p`.
But it can be `h2` or `h3` on design. 🤔